api/base/serializers.py

```diff
@@ -900,7 +900,10 @@ def to_representation(self, obj):
             else:
                 ret[name] = url
         if hasattr(obj, 'get_absolute_url') and 'self' not in self.links:
-            ret['self'] = self.extend_absolute_url(obj)
+            extended_url = self.extend_absolute_url(obj)
+            ret['self'] = extended_url
+            if 'info' in ret:
+                ret['info'] = extended_url
```

Comment:
@fabmiz Is there a good way to extend the info url rather than replacing it with the self url? If not, verify that the previous self url matches the info url before replacing it, in case someone changes how the info urls work in the future.